bokehjs/src/lib/models/glyphs/bezier.ts

```diff
@@ -173,7 +179,16 @@ export class Bezier extends Glyph {
   static init_Bezier(): void {
     this.prototype.default_view = BezierView
 
-    this.coords([['x0', 'y0'], ['x1', 'y1'], ['cx0', 'cy0'], ['cx1', 'cy1']])
+    this.define<Bezier.Props>({
+      x0:  [ p.XCoordinateSpec, {field: "x0"}  ],
+      y0:  [ p.YCoordinateSpec, {field: "y0"}  ],
+      x1:  [ p.XCoordinateSpec, {field: "x1"}  ],
+      y1:  [ p.YCoordinateSpec, {field: "y1"}  ],
+      cx0: [ p.XCoordinateSpec, {field: "cx0"} ],
+      cy0: [ p.YCoordinateSpec, {field: "cy0"} ],
+      cx1: [ p.XCoordinateSpec, {field: "cx1"} ],
+      cy1: [ p.YCoordinateSpec, {field: "cy1"} ],
```

Comment:
I'm sure you don't need to be told but the *cx*, *cy* values are not actually coordinates. Lumping them in with the coordinates in the first place was just a hacky shortcut. Is this an opportunity to improve the naming around these? (Is it worth it?)